backend/sql/postgresql/models/models.go

```diff
@@ -1776,24 +1776,15 @@ func (r *RetryPolicy) FromDto(dto *mgmtv1alpha1.RetryPolicy) {
 }
 
 type AccountOnboardingConfig struct {
-	HasCreatedSourceConnection      bool `json:"hasCreatedSourceConnection"`
-	HasCreatedDestinationConnection bool `json:"hasCreatedDestinationConnection"`
-	HasCreatedJob                   bool `json:"hasCreatedJob"`
-	HasInvitedMembers               bool `json:"hasInvitedMembers"`
+	HasCompletedOnboarding bool `json:"hasCompletedOnboarding"`
```

Comment:
This is fine and shouldnt be a breaking change, however it may be worth thinking through the idea that if someone has already completed this v1 of onboarding, we it could be worth somehow marking this new one as true automatically.

We could also say fuck that are just require everyone see the new onboarding guide to keep it simple too.

Reply:
Otherwise these DTO updates are just fine because the old properties will default to false if unspecified.